redis/example/kustomization.yaml

```diff
@@ -2,14 +2,30 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-  - ssh://github.com/utilitywarehouse/shared-kustomize-bases//redis/manifests/dev-enablement/redis-shared?ref=v0.0.0
+  - ssh://github.com/utilitywarehouse/shared-kustomize-bases//redis/manifests?ref=v0.0.0
```

Comment:
Wondering if we could reference also our own dev-ena deployment as an example. As we don't really have a way of testing this actually works

Reply:
I will add it as reference if you insist, but I would prefer to not go this path. 
- `Example` can be installed and it really works after just replacing the namespace,
- We should not assume it will be broken soon but rather try to keep it running,
- Our code will work, but can get outdated,
- Anyone can look for "shared-kustomize-bases" string in k8s-manifests anyway